src/libraries/Microsoft.PowerFx.Core/Binding/BindingConfig.cs

```diff
@@ -13,13 +13,15 @@ namespace Microsoft.PowerFx.Core.Binding
     [ThreadSafeImmutable]
     internal class BindingConfig
     {
-        public static readonly BindingConfig Default = new BindingConfig();
+        public static readonly BindingConfig Default = new BindingConfig(Features.None);
+
+        public Features Features { get; }
```

Comment:
I think this is mixing a global config (features) with a per-expression config (BindingConfig). May be worth passing as separate args to checkTypes, if that's what this is w.r.t.